Sources/Main/main.swift

```diff
@@ -2,5 +2,5 @@ import Server
 
 let parser = Parser()
 let response = Response(parser: parser)
-let server = Server(port: 3333, response: response)
+let server = Server(port: 5000, response: response)
```

Comment:
Now that you've had to change this, you can see that hard-coding it makes your server less amenable to its environment. For future reference, here's an article about parsing command-line arguments: https://www.hackingwithswift.com/articles/44/apple-s-new-utility-library-will-power-up-command-line-apps.